server/ai_http.go

```diff
@@ -210,10 +214,12 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 
 		// Prepare request to worker
 		controlUrlOverwrite := overwriteHost(h.node.LiveAITrickleHostForRunner, controlUrl)
+		eventsUrlOverwrite := overwriteHost(h.node.LiveAITrickleHostForRunner, eventsUrl)
 		workerReq := worker.LiveVideoToVideoParams{
 			ModelId:      req.ModelId,
 			PublishUrl:   overwriteHost(h.node.LiveAITrickleHostForRunner, subUrl),
 			SubscribeUrl: overwriteHost(h.node.LiveAITrickleHostForRunner, pubUrl),
```

Comment:
Unrelated but would you mind creating helper vars for these too? I don't get why they are done differently 🤔 kinda bothers me lol